planning/behavior_path_planner_common/include/behavior_path_planner_common/interface/scene_module_interface.hpp

```diff
@@ -259,6 +259,8 @@ class SceneModuleInterface
     return is_waiting_approval_ || current_state_ == ModuleStatus::WAITING_APPROVAL;
   }
 
+  virtual bool isRootLaneletToBeUpdated() { return false; }
```

Comment:
This method can be `const` ?

Reply:
@rej55 Thanks for the suggestion, I have added it in the latest commit